aws_lambda_powertools/shared/headers_serializer.py

```diff
@@ -123,3 +123,12 @@ def serialize(self, headers: Dict[str, Union[str, List[str]]], cookies: List[Coo
                 payload["headers"][key] = values[-1]
 
         return payload
+
+
+class NoopSerializer(BaseHeadersSerializer):
+    """
+    Noop serializer that doesn't do anything. This is useful for resolvers that don't need to set headers or cookies.
+    """
+
+    def serialize(self, headers: Dict[str, Union[str, List[str]]], cookies: List[Cookie]) -> Dict[str, Any]:
+        return {}
```

Comment:
you probably ain't gonna need it as you override build.

Reply:
Removed